app/helpers/groups_helper.rb

```diff
@@ -20,6 +20,14 @@ def new_event_button
     end
   end
 
+  def export_events_ical_button
```

Comment:
Unfortunately, I did not find how you want this helper tested. I'll add a test if you can provide me this information.